arangod/Aql/GraphNode.h

```diff
@@ -172,7 +172,8 @@ class GraphNode : public ExecutionNode {
   /// @brief the edge collection names
   std::vector<std::unique_ptr<aql::Collection>> _edgeColls;
 
-  /// @brief the vertex collection names
+  /// @brief the vertex collection names (can also be edge collections)
+  /// as an edge can also point to another edge, instead of a vertex.
```

Comment:
The closing parenthesis should be at the end of the second line:

```
  /// @brief the vertex collection names (can also be edge collections
  /// as an edge can also point to another edge, instead of a vertex).
```

Reply:
just fyi: You are able to add suggestions here, which can then be accepted. See https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Reply:
GitHub does not allow you to suggest changes for multi-line comments unfortunately, so such changes (involving 2+ lines) can only be suggested line by line, which is a bit stupid. I guess I could had just committed the change directly - sorry!